source/json/people.schema.json

```diff
@@ -9,8 +9,7 @@
         "chance": "name"
       },
       "role": {
-        "type": "string",
-        "pattern": "Staff|OWF Host|Partner|Hive NYC Partner|Advisor|Member, MLN Advisory Group"
```

Comment:
@gideonthomas @alanmoo – I had to remove this pattern to get the Django-returned JSON to validate. Is `role` being treated as a freeform string in Django? If so, is that going to be problematic for any future front-end functionality you're aware of, @alanmoo ?